server/index.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const db = require('../database');
+const generateEvents = require('../database/dummyData/userEventData.js');
+
+const app = express();
+app.use(bodyParser.json());
+
+app.get('/', (req, res) => {
+  db.selectAllEvents((err, data) => {
+    if (err) {
+      res.sendStatus(500);
+    } else {
+      res.status(200).send(data);
+    }
+  });
+});
+
+app.get('/userEventData', (req, res) => {
```

Comment:
Is there a reason this is a get request instead of a post request? It seems like this is generating data to put into your database (post-like) as opposed to querying your database to return data with the response (get-like).